source/MRMesh/MREndMill.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "MRExpected.h"
+#include "MRMeshFwd.h"
+
+namespace Json { class Value; }
+
+namespace MR
+{
+
+/// end mill cutter specifications
+struct EndMillCutter
+{
+    /// cutter type
+    enum class Type
+    {
+        /// flat end mill
+        Flat,
+        /// ball end mill
+        Ball,
```

Comment:
spherical? (if "Ball" is standard name - lets keep it)

Reply:
Yes, 'ball end mill' is a common name.